src/net/smooth/index.ts

```diff
@@ -175,12 +186,12 @@ export default function(
           segmentInfos: initSegmentInfos,
         });
       }
-      if (__DEV__) {
-        assert(responseData instanceof ArrayBuffer);
-      }
-      const responseBuffer = new Uint8Array(responseData as ArrayBuffer);
+      const responseBuffer = response.responseData instanceof Uint8Array
+      ? response.responseData
+       : new Uint8Array(response.responseData);
```

Comment:
some weird indent you got there

other than that [I'm good](https://www.youtube.com/watch?v=ftSUchAdVTE)